src/AutoRest.CSharp/Common/Output/Models/Types/ObjectTypeConstructor.cs

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
```

Comment:
Strange diff to include.

Reply:
I just did a remove all unused usings in a bunch of files i was touching.  Happy to undo this if you want

Reply:
That's a fine reason.